config/src/sumeragi.rs

```diff
@@ -90,7 +90,15 @@ impl ConfigurationProxy {
         self.trusted_peers = Some(TrustedPeers { peers });
     }
 }
+
 impl Configuration {
+    /// Set `trusted_peers` configuration parameter. Will overwrite
+    /// existing `trusted_peers` but does not check for duplication.
```

Comment:
It's an old method that's used nowhere, so maybe it would be more correct to just delete it? But what the doc meant I suppose is that it doesn't warn the user if some of the provided peers were identical.

Reply:
I don't mind deleting this method if it doesn't used anywhere.